main.go

```diff
@@ -14,8 +14,9 @@ import (
 	userDb "go.keploy.io/server/v2/pkg/platform/yaml/configdb/user"
 	"go.keploy.io/server/v2/utils"
 	"go.keploy.io/server/v2/utils/log"
+
 	//pprof for debugging
-	//_ "net/http/pprof"
+	_ "net/http/pprof"
```

Comment:
Please comment it out, since its init function is called when you import this package.